packages/core/test/SignUp.ts

```diff
@@ -18,6 +18,10 @@ describe('Signup', function () {
     const maxUsers = 3
     const maxAttesters = 3
 
+    const oldUsername = 0
```

Comment:
These shouldn't be defined at the top level. Generally we want to avoid sharing variables or state between tests. This is an anti-pattern we have in a lot of our test files that we're slowly removing.